tests/forms_tests/tests/test_forms.py

```diff
@@ -3015,6 +3015,108 @@ class UserRegistration(Form):
             "</td></tr>",
         )
 
+        def test_help_text_aria_describedby(self):
+            """
+            aira-describedby is added to the widget and refers to the id of
+            the help_text.
+            """
+
+            class UserRegistration(Form):
+                username = CharField(max_length=10, help_text="e.g., user@example.com")
+                password = CharField(
+                    widget=PasswordInput, help_text="Wählen Sie mit Bedacht."
+                )
+
+            p = UserRegistration()
+            self.assertHTMLEqual(
+                p.as_ul(),
```

Comment:
Since django 4.2(?) There is now an `as_div` method to which is preferable to all of these. I think we should add a test for that template too. 


Edit: looks like the change will need to be made to the `div.html` template too.